docfx/docs/migratingfromv1.md

```diff
@@ -204,6 +218,21 @@ The cursor and focus system has been redesigned in v2 to be more consistent and
 * Set [View.CursorVisibility](~/api/Terminal.Gui.View.CursorVisibility.yml) to the cursor style you want to use.
 * Remove any overrides of `OnEnter` and `OnLeave` that explicitly change the cursor.
 
+## Button.Clicked Event Renamed
+
+The `Button.Clicked` event has been renamed `Button.MouseClick`
```

Comment:
I think this should be the `Accept` event right @BDisp?

In v1 Clicked was fired under both Keyboard Enter and Mouse Click.  So closest direct behavioural replacement for `Button.Clicked` would be to use the `Button.Accept` event.

Reply:
Correct. It's using the `View.Accept` event.

https://github.com/gui-cs/Terminal.Gui/blob/a661fcecf7e60b694649a4d93df781c7ef5952ee/Terminal.Gui/View/View.cs#L115